src/WingmanDataTrail/MetricsVariables/helpers/areArraysEqual.ts

```diff
@@ -0,0 +1,4 @@
+import { isEqual } from 'lodash';
```

Comment:
Nice to see that `lodash` is one of the dependenvies that's externalized & [handled with SystemJS](https://grafana.com/developers/plugin-tools/key-concepts/npm-dependencies#dependency-sharing-mechanism), so we can use it freely without worrying about its impact on our bundle size :)